lidar_clusterer:
  ros__parameters:
    props_radii: "{key1: 1, key2: 2, key3: 3}"